schema/omega-services.async.api.yaml

```diff
@@ -414,6 +545,222 @@ components:
           items:
             $ref: '#/components/schemas/GeneralDatedDescription'
 
+    ListAgentParameters:
+      type: object
+      properties:
+        type:
+          $ref: '#/components/schemas/AgentTypes'
+          description: Limits the list to one-or-more types of Agent. If omitted, all types of agents are included in the list.
+        authority-file:
+          $ref: '#/components/schemas/AuthorityFile'
+          description: Limits the list of Agent(s) to only those that are (or are not) also Authority Files. If omitted both authorative and non-authorative Agents are included in the list.
+        depository:
+          $ref: '#/components/schemas/Depository'
+          description: Limits the list of Agent(s) to only those that are (or are not) also a depository. If omitted both depository and non-depository Agents are included in the list.
```

Comment:
How do `authority-file` and `depository` work together? If I want all authority files, whether or not they are depositories, do I just leave `depository` out? If I leave it in as `false` will that exclude the authority files that are depositories?

Reply:
These parameters act as filters.

> If I want all authority files, whether or not they are depositories, do I just leave depository out?

Yes, exactly.

> If I leave it in as false will that exclude the authority files that are depositories?

If `authority-files: true` and `depository: false` then you will get only agents that are Authority Files and are **not** a Place of Deposit.

If `authority-files: true` and `depository: true` then you will get only agents that are Authority Files and are a Place of Deposit.

If `authority-files: false` and `depository: false` then you will get only agents that are **not** Authority Files and are **not** a Place of Deposit.

If `authority-files: false` and `depository: true` then you will get only agents that are **not** Authority Files and are a Place of Deposit.